src/en/news/blog/2022/centralized_logging/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Centralized Logging on Ceph Dashboard"
+date: "2022-11-23"
+author: "Nizamudeen A"
+image: "images/banner.jpg"
+tags:
+  - "ceph"
+  - "dashboard"
+  - "logs"
+  - "monitoring"
+  - "loki"
+  - "promtail"
+  - "grafana"
+  - "centralized logging"
+---
+
+_Disclaimer: This article proclaims Loki as a hero._
+
+## What are we trying to solve?
+
+Ceph, as a distributed system, usually generates a lot of logs. And these logs can be from mons, mgr, osds or any of the clients of ceph.
+
+Naturally, we look at the logs to get some detailed information regarding the events that happened throughout the cluster lifecycle.
+Logs are what we immediately look for if we find something unusual going on in the cluster. Or if you like reading, its a fun place to be 😜.
+In anycase, we'll want to check through logs from different clients and they are not available from a single source. Its all distributed as it is intended.
+
+## Enter Loki.
+
+Thankfully, the Loki we are dealing with is not an Asgardian God(imagine the chaos). Our Loki is a log aggregation system which is designed to store and
```

Comment:
/s/which/that/

or

s/system which is designed to store/system designed to store/